### PR TITLE
add the graham scan in go

### PR DESCRIPTION
**Pull request by Liikt**
compiled with `go version go1.11 linux/amd64` and has no errors

### CODE REVIEW

**Comment by leios** [comment]
Hey @Liikt We are currently waiting on a review for golang. I don't know if we have too much expertise in the language. If no one steps up, I'll do the review, but I might ask dumb questions because I don't know the best practices in golang.

Just letting you know what's happening with this PR and #403, #404, #405, #406

**Comment by Liikt** [comment]
Sure thing. I'd be happy to answer any questions.

File: contents/graham_scan/graham_scan.md [hunk 1]
```diff
@@ -22,8 +22,14 @@ We can find whether a rotation is counter-clockwise with trigonometric functions
 [import:36-38, lang:"javascript"](code/javascript/graham-scan.js)
 {% sample lang="py" %}
 [import:4-6, lang:"python"](code/python/grahamScan.py)
+{% sample lang="go" %}
+<<<<<<< HEAD
```
Comment (Gustorn) [inline contents/graham_scan/graham_scan.md]: This doesn't look right

Comment (Liikt) [reply contents/graham_scan/graham_scan.md]: ohh woops :D i'll remove them XD

**Pull request merged**